.gitignore

```diff
@@ -48,7 +48,7 @@ Vagrantfile
 # Configs
 *.hcl
 !command/agent/config/test-fixtures/*.hcl
-!command/server/test-fixtures/*.hcl
+!command/server/test-fixtures/*/*.hcl
```

Comment:
Should this be `**/*.hcl` instead of `*/*.hcl`?

Reply:
it should! beae728